### PR TITLE
Create relations chats to users  mirosvo

### PR DESCRIPTION
**Pull request by Mirosvo92**

### CODE REVIEW

**Comment by Mirosvo92** [comment]
[Create relations chats to users one to Many](https://trello.com/c/yiuZiM1V/29-create-relations-chats-to-users)

File: src/db/models/relations.ts [hunk 1]
```diff
@@ -1,10 +1,22 @@
 export const relations = (db: any): void => {
   db.chats.hasMany(db.comments, {
-    as: db.aliases.chats.comments,
```
Comment (wearethefuture-org) [inline src/db/models/relations.ts:3]: why alias was removed?

Comment (wearethefuture-org) [reply src/db/models/relations.ts:3]: Please revert It

File: src/db/models/relations.ts [hunk 2]
```diff
@@ -1,10 +1,22 @@
...
+    onDelete: 'cascade',
     field: 'chat_id',
     targetKey: 'id',
     foreignKey: 'chatId'
-  })
+  });
+  db.users.hasMany(db.chats, {
+    onDelete: 'cascade',
+    field: 'owner_id',
```
Comment (wearethefuture-org) [inline src/db/models/relations.ts:10]: Where is aliases?

**Pull request merged**